.github/workflows/test-build.yml

```diff
@@ -63,5 +73,6 @@ jobs:
         run: |
           conda install anaconda-client
           conda config --set anaconda_upload no --set solver libmamba
-          VERSION_FROM_GIT_TAG=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-)test conda build . -c mosek -c anaconda -c pvlib -c slacgismo -c conda-forge --numpy 1.22.0
-          echo "gitversion=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-)" >> $GITHUB_OUTPUT
+          tag_ver=$(git tag --list "v*[0-9]" --sort=version:refname | tail -1 | cut -c 2-)
+          VERSION_FROM_GIT_TAG="${tag_ver}test" conda build . -c mosek -c anaconda -c pvlib -c slacgismo -c conda-forge --numpy 1.22.0
```

Comment:
same as above, we are missing some channels that were newly added

Reply:
rebased